crates/profile/src/memory_usage.rs

```diff
@@ -35,6 +35,22 @@ impl MemoryUsage {
                 // Note: This is incredibly slow.
                 let alloc = unsafe { libc::mallinfo() }.uordblks as isize;
                 MemoryUsage { allocated: Bytes(alloc) }
+            } else if #[cfg(windows)] {
```

Comment:
How does this work? Wasn't it supposed to be `if cfg!(windows)`? Yet clearly, it compiled.

Reply:
The `cfg_if` macro expands this to a `#[cfg]`'d out block

Reply:
Oh, sorry, I didn't notice the macro.